tasks/Module 3/task_2.py

```diff
@@ -0,0 +1,24 @@
+"""My split function."""
+
+
+def my_split(source_string, separator=None):
+    """My implementation of split() function."""
+    if separator is None:
+        separator = ' '
+    if separator == '':
```

Comment:
I'd move it to line 6 as we exit if that's True. It's good practice to put the exit conditions at the very top of your function.